mmpose/models/detectors/base.py

```diff
@@ -87,7 +87,7 @@ def _parse_losses(losses):
 
         return loss, log_vars
 
-    def train_step(self, data_batch, optimizer, **kwargs):
+    def train_step(self, data_batch):
```

Comment:
The `optimizer` seems necessary for runner. https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/epoch_based_runner.py#L28-L32

Reply:
The docstring says

>  This argument is unused and reserved.

Then let's keep it

Reply:
Thanks for checking.

Reply:
Updated.